apps/microtvm/reference-vm/zephyr/pyproject.toml

```diff
@@ -0,0 +1,140 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+[tool.black]
+line-length = 100
+target-version = ['py36']
+include = '(\.pyi?$)'
+exclude = '''
+
+(
+  /(
+      \.github
+    | \.tvm
+    | \.tvm_test_data
+    | \.vscode
+    | \.venv
+    | 3rdparty
+    | build\/
+    | cmake\/
+    | conda\/
+    | docker\/
+    | docs\/
+    | golang\/
+    | include\/
+    | jvm\/
+    | licenses\/
+    | nnvm\/
+    | rust\/
+    | src\/
+    | vta\/
+    | web\/
+  )/
+)
+'''
+[tool.poetry]
+name = "incubator-tvm"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+packages = [
+    { include = "tvm", from = "python" },
+]
+
+[tool.poetry.dependencies]
+attrs = "^19"
+decorator = "^4.4"
+numpy = "~1.19"
+psutil = "^5"
+scipy = "^1.4"
+python = "^3.6"
+tornado = "^6"
+typed_ast = "^1.4"
```

Comment:
created an RFC to discuss this: https://discuss.tvm.apache.org/t/rfc-consolidating-tvm-python-dependencies/8329/7

my hope is that we use the RFC approach A2; but if not I will re-align this to whatever approach is chosen. Because the intent here is to create a reproducible VM (much like the CI), I want the package versions used here to be more specific than ones specified generally as TVM dependencies.

Reply:
@areusch , After reading the RFC, do we want to align with a one of the requirement.txt ?. So we can synergy between this and ci-qemu ? and would suggest introduction of poetry after next RFC if and when that happens, thus the changes will go here as well.

Reply:
@manupa-arm I definitely agree with you that the approach used here should mirror the one used in the Consolidating Python Dependencies RFC. I guess, in a perfect world i agree it would make sense to introduce `requirements.txt` here now, and `pypoetry.toml` later. but, it's not like there exists a `requirements.txt` to copy now, and this PR has been out for weeks now. i'd rather just merge this change now, and treat these two tasks as separate efforts. after we finish addressing the RFC, I will align the approach here with the one taken there. we still have not yet resolved whether to depend on poetry as a whole project yet; however, the VM, by contrast to the project, prefers to specify dependencies with more version constraints in the interest of maintaining a working build.